examples/todo-jwt/src/__tests__/acceptance/home-page.acceptance.ts

```diff
@@ -4,7 +4,7 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {Client} from '@loopback/testlab';
-import {TodoListApplication} from '../..';
+import {TodoListApplication} from '../../application';
```

Comment:
i believe it works without this change?

Reply:
It's better to revert the change. For tests, it's recommended to import from the package root without relative paths.